src/AutomatedCar/SystemComponents/Sensors/Sensor.cs

```diff
@@ -31,13 +32,65 @@ public Sensor(VirtualFunctionBus virtualFunctionBus)
             this.triangle = new SensorTriangle();
         }
 
+        protected abstract List<WorldObject> FilterRelevantWorldObjects();
+
+//        protected abstract void SaveWorldObjectsToPacket(List<WorldObject> worldObjects);
         protected abstract void SaveWorldObjectsToPacket();
 
-        protected abstract List<WorldObject> FilterRelevantWorldObjects();
+        protected AutomatedCar GetAutomatedCar()
```

Comment:
Ezt Egonnál is átbeszéltük, sztem ő már át is írta az implementációban, de nem kapjuk meg direktben a kocsit. Maga a kocsi a VFB-re írja fel a jelenlegi koordinátáit, és azt olvassa ki a szenzor.

Reply:
A WorldObjecteket is a World singletonból kérjük le a szenzorokban, ami tartalmazza az AutomatedCart is (tehát onnan is el lehet érni az autót), akkor van bármi előnye a VFB-re packetet írni a koordinátáknak? Úgy érzem, hogy ez felesleges bonyolítás.